samples/cpp/benchmark_app/main.cpp

```diff
@@ -378,11 +378,19 @@ int main(int argc, char* argv[]) {
         for (auto& device : devices) {
             auto& device_config = config[device];
 
+            ov::hint::PerformanceMode ov_perf_hint = ov::hint::PerformanceMode::THROUGHPUT;
+
             // high-level performance modes
             if (!device_config.count(ov::hint::performance_mode.name())) {
-                device_config.emplace(ov::hint::performance_mode(get_performance_hint(device, core)));
+                auto ov_perf_hint = get_performance_hint(device, core);
```

Comment:
I think the idea with `none` as -hint isn't suitable anymore. `ov::hint::performance_mode` is always set to a specific value now. Also we should change -hint default value to throughput. Because that is the actual default value now. @nkogteva I'm not aware of all the reasons for having `none`. What do you think?

Reply:
> I think the idea with `none` as -hint isn't suitable anymore. `ov::hint::performance_mode` is always set to a specific value now. Also we should change -hint default value to throughput. Because that is the actual default value now. @nkogteva I'm not aware of all the reasons for having `none`. What do you think?

1. Throughput is default value already accouring to BA logic.
2. I think we should keep BA behaviour as is with supported none value, but just stop pass this value into OpenVINO.

Reply:
1. We could make it more explicit by updating the default value for the cmd arg
2. Ok

Reply:
> 1. We could make it more explicit by updating the default value for the cmd arg
> 2. Ok

I have no objections to make it more explicit

Reply:
In current BA logic, `-hint none` is used to prevent users from setting hint and nstreams/nthreads/pin together.

This PR do not change this logic, but just stop pass `ov::hint::PerformanceMode::UNDEFINED` to OV Runtime

Reply:
Yes, please, keep `none`. Just replace "" with "throughput" as default -hint value

Reply:
@Wovchena Do you mean passing throughput hint and nstreams/nthreads setting together? This is not backward compatible. Not sure if all plugins can handle this.

Reply:
Hm. https://github.com/openvinotoolkit/openvino/pull/16563 claims
> * PerformanceHint's UNDEFINED value is blocked by VPUX plugin. Once VPUX plugin is updated, we can make change UNDEFINED = LATENCY

So we should keep empty value to allow UNDEFINED. Ok. Sorry, my bad